File: Jmo.Domain/Question.cs
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;

namespace Jmo.Web.Data.Domain
{
    public class Question : IEntity
    {
        public int Id { get; set; }

        [StringLength(2500)]
        public string Questionant { get; set; }

        [StringLength(250)]
        public string ImagenUrl { get; set; }

        public string AnswerRestrospective { get; set; }

        public int CategoryId { get; set; }
        public Category Category { get; set; }

        public ICollection<Choise> Choises { get; set; }

        public string ImageFullPath => string.IsNullOrEmpty(ImagenUrl)
           ? "https://braingameschallenges.azurewebsites.net//images/noimage.png"
           : $"https://braingameschallenges.azurewebsites.net{ImagenUrl.Substring(1)}";

        public Question()
        {
            Choises = new Collection<Choise>();
        }
    }
}

File: jmo_flutapp/lib/models/question.dart
class Question{

 int _id ;

 String _questionant ;

 String _imagenUrl;

 String _answerRestrospective ;

 int _categoryId ;

// Category _category ;

// List<Choise> _choises ;

Question(this._questionant,this._imagenUrl,this._answerRestrospective, this._categoryId);
Question.WithId(this._id, this._questionant,this._imagenUrl,this._answerRestrospective, this._categoryId);

int get id=>_id ;

String get questionant =>_questionant ;

String get imagenUrl =>_imagenUrl;

String  get answerRestrospective =>_answerRestrospective ;

int get categoryId =>_categoryId ;

set questionant(String value){
  _questionant=value;
}

set imagenUrl(String value){
  _imagenUrl=value;
}

set answerRestrospective(String value){
  _answerRestrospective=value;
}

set categoryId(int value){
  _categoryId=value;
}

Map<String,dynamic> toMap(){
  var map = Map<String,dynamic>();
  map["questionant"] = _questionant;
  map["imagenUrl"] = _imagenUrl;
  map["answerRestrospective"] = _answerRestrospective;
  map["categoryId"] = _categoryId;
  if(_id != null){
    map["id"]=_id;
  }
  return map;
}

Question.fromObject(dynamic o)
{
  this._questionant = o["questionant"]   ;
  this._imagenUrl = o["imagenUrl"]   ;
  this._answerRestrospective = o["answerRestrospective"] ;
  this._categoryId = o["categoryId"] ;
  this._id = o["id"];
}



}